docs/research/user-experience/xmr-btc-user-interface-definition.md

```diff
@@ -9,13 +9,16 @@ This document will be used as a basis for defining and evaluating low-fidelity U
 
 This section sums up assumptions around the current setup, protocol and existing software. 
 
-### Technical Limitations
+### Security Assumptions
 
-Swap-solutions that run completely in the browser (webpage) can be problematic because blockchain nodes, notably [bitcoind](https://github.com/bitcoin/bitcoin/issues/11833), do not specifying [CORS headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) in their response.
+Swap-solutions that run completely in the browser (webpage) and rely on connecting to a locally running blockchain node are currently not possible because specifying [CORS headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) is a security risk.
 This means the browser will reject a response from a locally running Bitcoin node.
+Discussions have been ongoing, notably for [bitcoind](https://github.com/bitcoin/bitcoin/issues/11833), to ad features that would allow webpages to communicate with locally running blockchain nodes directly.
```

Comment:
```suggestion
Discussions have been ongoing, notably for [bitcoind](https://github.com/bitcoin/bitcoin/issues/11833), to add features that would allow webpages to communicate with locally running blockchain nodes directly.
```